crates/oxc_linter/src/fixer/fix.rs

```diff
@@ -85,6 +85,22 @@ impl FixKind {
     pub fn can_apply(self, rule_fix: Self) -> bool {
         self.contains(rule_fix)
     }
+    pub(crate) fn emoji(self) -> &'static str {
+        if self.is_dangerous() {
+            return "⚠️";
+        } else if self.is_empty() {
+            return "";
+        }
+        match self {
+            Self::Fix => "🛠️",
+            Self::Suggestion => "💡",
+            Self::Dangerous => "⚠️",
+            _ => {
+                debug_assert!(false, "Please add an emoji for FixKind: {:?}", self);
```

Comment:
i know this Pr is still a draft, but wouldn't an exhaustive match be better here? it means you can assert this at compile time rather than debug runtime?

Reply:
To be exhaustive, I'd need to have branches for all valid `u8` values. However, there are only ~6 valid `FixKind` values.

Reply:
ah gotcha